scripts/build_provenance

```diff
@@ -4,14 +4,25 @@ readonly SCRIPTS_DIR="$(dirname "$0")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
-readonly BUILD_COMMAND=$1
-readonly OUTPUT_PATH=$2
+readonly OUTPUT_PATH=$1
 
-eval "./scripts/docker_run ${BUILD_COMMAND}"
+# Shift arguments to omit the output path. The remaining arguments are the
+# build arguments.
+shift
+
+eval "./scripts/docker_run ${@}"
```

Comment:
I'm surprised this works with the the quotes, does eval do its own splitting / parsing? Or maybe docker_run does

Reply:
Bash arg handling is very confusing, but: In `docker_run` we use `$*` when invoking the docker CLI. Afaik `$*` effectively means args a delineated by whitespace. Hence passing as args as a joined string doesn't make difference.

Reply:
An upside of the code I wrote in this for formatting bash args into a toml string is that it doubles as a handy tool for investigating bash arg handling :p

Reply:
docker_run concats args into a single string by default. That string is passed to `fix_docker_user_and_run`, which executes that string using `su --session-command`. I think thats where the splitting happens?